### ISSUE TITLE
Error visiting page when using NEXT_PUBLIC_DRUPAL_BASE_URL if port number is specified

### ISSUE THREAD

**Issue by 2dareis2do** [chapter-three/next-drupal#476]
The issue I have is when visiting subpage with a port number I get the error 

e.g if I visit http://localhost:3001/recipes/super-easy-vegetarian-pasta-bake

from browser:

'Server Error
Error: Not Found
This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
<unknown>

file:///Users/myuser/Sites/next-example-umami3/node_modules/next-drupal/dist/index.js (2550:19)
processTicksAndRejections

node:internal/process/task_queues (96:5)'

form termoinal

`error - Error: Not Found
    at /Users/myuser/Sites/next-example-umami3/node_modules/next-drupal/dist/index.js:2550:19
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  type: 'JsonApiErrors',
  errors: 'Not Found',
  statusCode: 404,
  page: '/[...slug]'`.

In other words I have to swap out say `http://umami3.internal:80` for http://umami3.internal`.  Then I can visit an article or recipe with out issue.
